src/Common/REST/TEC/V1/Parameter_Types/Definition_Parameter.php

```diff
@@ -99,6 +100,53 @@ public function get_sanitizer(): Closure {
 		return $this->sanitizer ?? fn( $value ): array => $this->sanitize( $value );
 	}
 
+	/**
+	 * Filters the data.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $data The data to filter.
+	 *
+	 * @return array The filtered data.
+	 *
+	 * @throws InvalidRestArgumentException If the data is invalid.
+	 */
+	public function filter( array $data = [] ): array {
+		$collections = $this->get_collections();
+
+		$filtered_data = [];
+
+		/** @var PropertiesCollection $collection */
+		foreach ( $collections as $collection ) {
+			/** @var Parameter $property */
+			foreach ( $collection as $property ) {
+				$param_name = $property->get_name();
+				$argument   = $this->get_name() ? $this->get_name() . '.' . $param_name : $param_name;
```

Comment:
Minor: I find it easier to read interpolation, especially in the context of a ternary.

```suggestion
				$argument   = $this->get_name() ? "{$this->get_name()}.{$param_name}" : $param_name;
```